xclim/core/indicator.py

```diff
@@ -156,20 +187,29 @@ class Indicator:
     # Number of input DataArray variables. Should be updated by subclasses if needed.
     _nvar = 1
 
-    # Allowed metadata attributes on the output
-    _cf_var = [
+    # Allowed metadata attributes for the variables
+    _var_names = [
```

Comment:
I think `_var_names` is confusing. I thought it referred to the names of the output DataArrays.